app/javascript/mastodon/features/account/components/account_note.js

```diff
@@ -57,8 +59,15 @@ class AccountNote extends ImmutablePureComponent {
     value: null,
     saving: false,
     saved: false,
+    editable: false,
   };
 
+  constructor (props) {
+    super(props);
+    this.setEditable = this.setEditable.bind(this);
+    this.unEditable = this.unEditable.bind(this);
+  }
```

Comment:
In the remaining of the code, we don't use this, but `functionName = () => { … }`

Reply:
Ah, yes, OK, arrow function is better, right?

Reply:
I think so, at least it would be more consistent with the codebase!